addon/components/o-s-s/badge.ts

```diff
@@ -38,9 +38,9 @@ interface OSSBadgeArgs {
   icon: string;
   image: string;
   text: string;
-  plain: boolean;
-  size: SizeType;
-  skin: SkinType;
+  skin?: SkinType;
```

Comment:
skin is not optional as reflectected by the assertion in the constructor.

Reply:
In the constructor, it is just @icon, @image or @text
```typescript
const contentArguments = [args.icon, args.image, args.text].filter((arg: string) => arg);
```

The skin is only used in `get skinClass()` function with
```typescript
const skin: SkinType | undefined = this.args?.skin;
if (!skin) return '';
```
So I deduce that it is optional, I can change that if it is not clear

Reply:
My bad, the assertion I was talking about is not in the constructor but in the sizeClass getter.